tests/System.Azure.Experimental.Tests/System.Azure.Experimental.Tests.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\tools\common.props" />
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
```

Comment:
Hmm... this is a known and recurring issue. I will make an effort to reach out to someone on the VS (or possibly xunit) team to figure out why the test discovery is still flaky for a large solution like this with mix of different netcoreapp versions.